libs/features/mas/src/aem-fragment.js

```diff
@@ -244,18 +223,15 @@ export class AemFragment extends HTMLElement {
     }
 
     get data() {
-        if (this.#data) return this.#data;
```

Comment:
this was meant to prevent re-transforming data on each `aemFragment.data` calls.

Reply:
maybe not needed, up to you.